src/coreclr/src/jit/emit.cpp

```diff
@@ -5515,33 +5515,33 @@ UNATIVE_OFFSET emitter::emitFindOffset(insGroup* ig, unsigned insNum)
  *  block.
  */
 
-UNATIVE_OFFSET emitter::emitDataGenBeg(UNATIVE_OFFSET size, UNATIVE_OFFSET alignment)
+UNATIVE_OFFSET emitter::emitDataGenBeg(unsigned size, unsigned alignment, var_types dataType)
 {
     unsigned     secOffs;
     dataSection* secDesc;
 
     assert(emitDataSecCur == nullptr);
 
-    // The size must not be zero and must be a multiple of 4 bytes
-    // Additionally, 4 bytes is the minimum alignment that will
+    // The size must not be zero and must be a multiple of MIN_DATA_ALIGN
+    // Additionally, MIN_DATA_ALIGN is the minimum alignment that will
     // actually be used. That is, if the user requests an alignment
     // of 1 or 2, they will get  something that is at least 4-byte
     // aligned. We allow the others since 4 is at least 1/2 and its
```

Comment:
You have changed this comment so that it's now using a mix of "4" and MIN_DATA_ALIGN. I'd reword the rest of the comment as well.
```suggestion
    // actually be used. That is, if the user requests an alignment
    // less than MIN_DATA_ALIGN, they will get  something that is at least MIN_DATA_ALIGN
    // aligned. We allow smaller alignment to be specified since it is
```